torch/fx/experimental/symbolic_shapes.py

```diff
@@ -113,6 +113,20 @@ def has_hint(a):
         return a.node.has_hint()
     return True
 
+def is_concrete_int(a: Union[int, SymInt]):
+    r""" Utility to check if underlying object
+    in SymInt is concrete value. Also returns
+    true if integer is passed in.
+
+    Args:
+        a (SymInt or int): Object to test if int
+    """
+    assert isinstance(a, SymInt) or isinstance(a, int)
+    if isinstance(a, SymInt):
+        return a.node.is_int()
```

Comment:
This is wrong, a SymInt will always return True here.

Reply:
What I am trying to test here is if the backing value for SymInt is a concrete integer, instead of a symbol.

Reply:
yes, I think what I'd do is peak at the sympy expr and see if it's Integer